modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -22,83 +22,339 @@ Runtime Fabric enables you to specify the level of severity of the message writt
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+    Name        splunk
+    Match       *
+    Host        127.0.0.1
+    Port        8088
+    TLS         On
+    TLS.Verify  Off
+    Message_Key my_key
+```
+
+==== Syslog Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default value
+| Match | Specifies the pattern to match for tags of logs to be outputted by this plugin. | |
+| Host| Specifies the IP address or hostname of the target syslog server. | |	127.0.0.1
+| Port | Specifies the port on which your syslog server is listening. | | 514
+| Mode | Specifies the protocol to use ( udp, tcp or tls). | | udp
+| syslog_format | Specifies the syslog format: `rfc3164` or `rfc5424`. | | rfc5424
+| syslog_maxsize | Specifies the maximum message size. | | 2048 for rfc5424 or 1024 for rfc3164
+| syslog_severity_key | Specifies the key to use for the syslog severity. | | 6 (info)
+| syslog_facility_key | Specifies the key to use for the syslog facility. | | 1 (user)
+| syslog_hostname_key | Specifies the key to use for the name of the host. | |
+| syslog_appname_key | Specifies the key to use for the application name. | |	
+| syslog_procid_key | Specifies the key to use for the process identifier. | |	
+| syslog_msgid_key | Specifies the key to use for message ID. | |	
+| syslog_message_key | Specifies the key to use for the syslog message.	| |	
+| syslog_sd_key | Specifies the key to syslog structured data, which can be set multiple times. This must be a map. | |
+|===
 
+Example Fluent Bit output plugin configuration:
+```
+[OUTPUT]
+	Name   syslog
+	Match  *
+
+	Host   127.0.0.1
+	Port   514
+	Mode   tcp
+
+	Syslog_Format        rfc5424
+	Syslog_Severity_key  serverity
+	Syslog_Facility_key  factility
+	Syslog_Hostname_key  hostname
+	Syslog_Appname_key   app
+	Syslog_Procid_key    pid
+	Syslog_Message_key   log
+	Syslog_sd_key        info=]1
+	Syslog_sd_key        info2
+```
+
+== Disable Log Forwarding for Specific Pods
+
+By default, Fluent Bit log forwarding is enabled on all pods (MuleApplication pods, Runtime Fabric pods and Infrastructure pods). 
+
+To disable log forwarding for specific pods, you can annotate it with `fluentbit.io/exclude`, as shown in the following example:
+```
+apiVersion: v1
+kind: Pod
+metadata:
+  name: apache-logs
+  labels:
+    app: apache-logs
+  annotations:
+    fluentbit.io/exclude: "true"
+spec:
+  containers:
+  - name: apache
+  ...
+```
+OR, you can use the following kubectl command to annotate it on pods in a specific namespace:
+```
+kubectl annotate pod <pod-name> -n <namespace> fluentbit.io/exclude="true"
+```
+Note that the annotation value is boolean and must be in quotation marks. 
```

Comment:
FYI @hegderohit89. Let's remove this since we don't want customer to edit the pod configuration or run kubectl commands. 
Instead, we can document that the Application level configuration is not supported yet, and will be provided in future version. Once provided, A user can enable/disable application level log forwarding for the external log forwarding service.

Reply:
Removed this